corehq/apps/case_importer/views.py

```diff
@@ -404,6 +405,30 @@ def excel_commit(request, domain):
     except ImporterError as e:
         return render_error(request, domain, get_importer_error_message(e))
 
+    if not request.POST.get('confirm'):
+        num_cases = 0
+        worksheet_titles = _get_workbook_sheet_names(case_upload)
+        for i in range(len(worksheet_titles)):
+            with case_upload.get_spreadsheet(i) as spreadsheet:
+                num_cases += spreadsheet.max_row
+        THRESHOLD = 100 * 1000
```

Comment:
as part of this, can we check for a large 1D excel file upload? eg...one column, but many rows?